docs/source/conf.py

```diff
@@ -20,23 +20,11 @@
 # -- Project information -----------------------------------------------------
 
 project = "responder"
-copyright = "2018, A Kenneth Reitz project"
+copyright = "2018-2024, A Kenneth Reitz project"
```

Comment:
i typically just add the current year and keep it up to date

Reply:
Updated.